packages/browser-wallet/src/popup/shared/utils/transaction-helpers.ts

```diff
@@ -91,7 +92,10 @@ export function validateBakerStake(
     const bakerStakeThreshold = chainParameters?.minimumEquityCapital.microCcdAmount || 0n;
     const amount = ccdToMicroCcd(amountToValidate);
 
-    if (bakerStakeThreshold > amount) {
+    const amountNotChanged =
+        accountInfo?.type === AccountInfoType.Baker && amount === accountInfo.accountBaker.stakedAmount.microCcdAmount;
+
+    if (!amountNotChanged && bakerStakeThreshold > amount) {
```

Comment:
The double negation here makes this harder to understand than if the variable was just `amountChanged`.